src/components/Chart/ChartOptions.jsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React, { useEffect, useState } from 'react';
+import Dropdown from 'react-bootstrap/Dropdown';
+import Form from 'react-bootstrap/Form';
+import SelectableContext from 'react-bootstrap/SelectableContext';
+import { useDispatch, useSelector } from 'react-redux';
+import { NumberInlineInput } from 'pc-nrfconnect-shared';
+import { func, number } from 'prop-types';
+
+import { chartState, setYMax, setYMin } from '../../reducers/chartReducer';
+
+const unitLabels = ['\u00B5A', 'mA', 'A'];
```

Comment:
For constants that will not change I prefer using capital naming, not a requirement though.

```suggestion
const UNIT_LABELS = ['\u00B5A', 'mA', 'A'];
```